tools/build.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import argparse
+from setup import get_build_dir
+import subprocess
+import os
+import time
+
+project_root = get_build_dir() / '..'
+
+def touch_cmake_lists():
+    cmake_lists_path =  project_root / 'CMakeLists.txt'
+    os.utime(cmake_lists_path, times=None)
+
+build_timestamp_path = project_root / '.build_timestamp'
```

Comment:
Is it possible to *not* use this file at all, and instead `touch` every time when the `.build_sources` mismatches with the real sources?